osu.Game/Overlays/Toolbar/ToolbarButton.cs

```diff
@@ -78,9 +78,8 @@ public ToolbarButton()
                 HoverBackground = new Box
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Colour = OsuColour.Gray(80).Opacity(180),
+                    Colour = OsuColour.Gray(80).Opacity(0),
```

Comment:
Yeah I'm not sure about this one, either. Splitting flash & hover entirely into separate boxes would definitely be cleaner.

Reply:
Agree with this. Splitting out would be best.